packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1524,6 +1526,9 @@ export const createAzureCommunicationCallAdapterFromClient: (
   options?
 ): Promise<CallAdapter> => {
   const deviceManager = (await callClient.getDeviceManager()) as StatefulDeviceManager;
+  deviceManager.getCameras();
```

Comment:
I can't remember, does this trigger the camera permissions browser popup?

Reply:
(if so we can't call this in the adapter constructor)

Reply:
I tried it on incognito mode and didn't see any pop up aside from call readiness modals

Reply:
For skipping config screen and joining immediately should it not skip the call readiness modals?

Reply:
What's the reason we need to call this in the constructor also? Could we call it on joinCall if cameraOn is true?

Reply:
Tried incognito mode without skipping config screen and the only pop up I see is call readiness. When config screen is skipped and I go into the call without permissions set, I see this 
<img width="1728" alt="Screenshot 2023-06-30 at 2 21 11 PM" src="https://github.com/Azure/communication-ui-library/assets/96077406/329ede87-c716-415b-9b79-f5f0e4e4dce0">
When permissions are set before going into the call, I see no pop up
I think this  should be expected? 
The reason we call getCameras and getMics here is because we found that before, we are getting device info at the middle of call connecting,  in this way we can get the devices at the correct time

Reply:
this should yes, something to test @carocao-msft 👀

Reply:
That sounds right re the permissions thanks for checking!
We don't await the getDevices calls so there's no guarantee we will have device information and it could still popup during the connecting phase.
Perhaps best to await them all here (use await Promise.all to avoid sequential waiting)?

Reply:
(sadly we can't call it on joinCall becuase it's not async fn :/)

Reply:
Updated :)